modules/core/src/main/java/org/apache/ignite/hlc/HybridClock.java

```diff
@@ -17,59 +17,22 @@
 
 package org.apache.ignite.hlc;
 
-import java.time.Clock;
-import org.apache.ignite.internal.tostring.S;
-
 /**
  * A Hybrid Logical Clock.
  */
-public class HybridClock {
-    /** Latest timestamp. */
-    private HybridTimestamp latestTime;
-
-    /**
-     * The constructor which initializes the latest time to current time by system clock.
-     */
-    public HybridClock() {
-        this.latestTime = new HybridTimestamp(Clock.systemUTC().instant().toEpochMilli(), 0);
-    }
-
+public interface HybridClock {
```

Comment:
Any reason to have this interface ? I can't imaging more than one implementation.

Reply:
Fixed. Seems that it doesn't need now.